### PR TITLE
chore: remove sys.path override

### PR DESCRIPTION
**Pull request by jfuss**
*Issue #, if available:*
#1911 

*Description of changes:*
Remove `sys.path` overrides that were done to support a testing script.

*Description of how you validated changes:*
Remove `sys.path` overriding.

*Checklist:*

- [ ] Add/update [unit tests](https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#unit-testing-with-multiple-python-versions) using:
    - [ ] Correct values
    - [ ] Bad/wrong values (None, empty, wrong type, length, etc.)
    - [ ] [Intrinsic Functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html)
- [ ] Add/update [integration tests](https://github.com/aws/serverless-application-model/blob/develop/INTEGRATION_TESTS.md)
- [ ] `make pr` passes
- [ ] Update documentation
- [ ] Verify transformed template deploys and application functions as expected

*Examples?*

Please reach out in the comments, if you want to add an example. Examples will be 
added to `sam init` through https://github.com/awslabs/aws-sam-cli-app-templates/

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: samtranslator/feature_toggle/feature_toggle.py [hunk 1]
```diff
@@ -13,9 +13,6 @@
 )
 from samtranslator.metrics.method_decorator import cw_timer
 
-my_path = os.path.dirname(os.path.abspath(__file__))
```
Comment (mndeveci) [inline samtranslator/feature_toggle/feature_toggle.py:16]: Why we needed this before?

**Review by mndeveci — APPROVED** [review]

File: samtranslator/feature_toggle/feature_toggle.py (hunk 1 above)
Comment (jfuss) [reply samtranslator/feature_toggle/feature_toggle.py:16]: @mndeveci See discussion here: https://github.com/aws/serverless-application-model/pull/2076#discussion_r796798622

TLDR: Was due to the test script. Since this is clobbering things on customers machines, I removed feature_toggle from the test script. If we really need it there (which I don't think we do), we can find a different way to add it back.

**Review by hawflau — APPROVED** [review]
